woocommerce-services.php

```diff
@@ -390,6 +390,66 @@ public function pre_wc_init() {
 			$this->taxjar->init();
 		}
 
+		public function get_service_schema_defaults( $schema ) {
+			$defaults = array();
+
+			if ( ! property_exists( $schema, 'properties' ) ) {
+				return $defaults;
+			}
+
+			foreach ( get_object_vars( $schema->properties ) as $prop_id => $prop_schema ) {
+				if ( property_exists( $prop_schema, 'default' ) ) {
+					$defaults[ $prop_id ] = $prop_schema->default;
+				}
+
+				if (
+					property_exists( $prop_schema, 'type' ) &&
+					'object' === $prop_schema->type
+				) {
+					$defaults[ $prop_id ] = $this->get_service_schema_defaults( $prop_schema );
+				}
+			}
+
+			return $defaults;
+		}
+
+		public function init_core_wizard_shipping_config() {
+			$store_country = WC()->countries->get_base_country();
+
+			if ( 'US' === $store_country ) {
+				$country_method = 'usps';
+			} elseif ( 'CA' === $store_country ) {
+				$country_method = 'canada_post';
+			} else {
+				return; // Only set up live rates for US and CA
+			}
+			
+			$zones_to_setup = array(
+				'domestic' => 1, // TODO: this might be too fragile/assumptive
```

Comment:
If you want to be extra-safe, you can check that the only location on the zone is of type `country` and value `$store_country`, and bail otherwise.